src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/court/CourtEmailFactory.kt

```diff
@@ -382,6 +395,8 @@ object CourtEmailFactory {
           )
         }
       }
+
+      else -> null
```

Comment:
What other action could it be here require the null catch-all? Would the missing link email drop into here?